src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,105 @@
+package duke;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+import duke.storage.Storage;
+import duke.ui.Ui;
+import duke.parser.Parser;
+import duke.data.task.Task;
+import duke.data.TaskList;
+import duke.data.exception.DukeException;
+public class Duke {
+
+    private final Storage storage;
+    private TaskList tasklist;
```

Comment:
Should the names `tasklist` in the various files be refactored to `taskList` to follow camel case?